.github/workflows/pylint.yml

```
@@ -0,0 +1,16 @@
+name: Linter
+on: pull_request
+jobs:
+  PyLint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run checks
+        run: |
+          pip install --user $(egrep "pylint.*" ./requirements.txt)
```

```suggestion
          pip install --user -r <(grep "^pylint" ./requirements.txt)
```

(Using `-r` permits normal requirements file syntax, such as comments).

---

Fixed it locally.